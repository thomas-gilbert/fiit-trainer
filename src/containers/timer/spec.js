import React from 'react';
import renderer from 'react-test-renderer';
import Timer from './index.js';

describe('Timer Component', () => {
	it('renders correctly', () => {
	  const component = renderer.create(
	    <Timer />
	  ).toJSON();
	  expect(component).toMatchSnapshot();
	});
});