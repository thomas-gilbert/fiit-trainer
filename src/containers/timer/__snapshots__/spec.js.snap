// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer Component renders correctly 1`] = `
<div
  className="timer"
>
  <h2>
    Exercise Name
  </h2>
  <h1>
    COUNTDOWN
  </h1>
  <h2>
    Next Thing rest/exercsie
  </h2>
</div>
`;
